tests/atest.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+from concrete import fhe
+from Bio.Seq import Seq, MutableSeq
+
+import sys, os, time
+sys.path.append(os.getcwd())
+from concrete_biopython.FheSeq import FheSeq, FheMutableSeq
+from concrete_biopython.SeqWrapper import SeqWrapper
+from concrete_biopython.BioCircuit import BioCircuit
+
+
+
+def process_sequence(seq1, seq2):
+    """
+    Check wether two elements are identical
+    This is a dummy function for testing, we will redefine it for each application later
+    """
+    return seq1[0]==seq2[1]
+
+
+def compute_fhe_output(seq_list, process_seq, description, res=False):
+    """
+    Pack up the creation and running of a circuit,
+    and display the result along with the computation times
+    """
+    # First print the description and a waiting message
+    print(description)
+    print("Computing ...", end="", flush=True)
+    print("\r", end="")
+
+    # prepare configuration
+    configuration=fhe.Configuration(
+        enable_unsafe_features=True,
+        use_insecure_key_cache=True,
+        insecure_key_cache_location=".keys",
+        dataflow_parallelize=False,
+    )  
+
+    
+    # Create a BioCircuit wrapped circuit
+    circuit = BioCircuit(
+        function=process_seq,
+        len_seqs=[len(seq) for seq in seq_list],
+        configuration=configuration,
+    )
+
+    output = circuit.encrypt_run_decrypt_as_integers(*seq_list)
```

Comment:
How much difference are there between univariate and direct multiplication?

Reply:
no difference now, I don't know why there was any for me before

Reply:
Nice, so it's using the trick by default right?

Reply:
yes !